custom_components/tuya_local/devices/casdon_td_pro_2.yaml

```diff
@@ -0,0 +1,206 @@
+name: Casdon TD Pro 2
+products:
+  - id: wab6oustdukujpkn
+    name: 蒸烤箱TD-pro二代
+primary_entity:
+  entity: switch
+  icon: "mdi:power"
+  name: Switch
+  dps:
+    - id: 1
+      type: boolean
+      name: switch
+
+secondary_entities:
+  - entity: switch
+    name: Start
+    icon: "mdi:play-pause"
+    dps:
+      - id: 2
+        name: switch
+        type: boolean
+  - entity: number
```

Comment:
I think this may be better as a climate entity, as it seems most other ovens supported by HA expose themselves this way, and users expect a thermostat-like interface for their oven. The climate entity may cover dps 2 (hvac_mode), 7 (temperature) and 8 (current_temperature), and maybe also dp 105 (preset_mode).
You could also convert dp 102 values to "heating", "idle", "off" hvac_action, but since it would only give partial information, you need to also keep the sensor in that case.